docs/source/operations/configure_prerequisites.md

```diff
@@ -31,6 +31,11 @@ The above command generates an SSH key-pair: **gitops** (private key) and **gito
 
 Use the path to the private key (**gitops**) in the `gitops.private_key` section of the [configuration file](./corda_networkyaml.md).
 
+---
+**NOTE:** The **gitops**(private key) should have access permissions for the ansible host.
```

Comment:
"Ensure that the Ansible host has read-access to the private key file (**gitops**)."